pkg/util/timeutil/pgdate/field_extract.go

```diff
@@ -64,6 +64,8 @@ type fieldExtract struct {
 	tzSign int
 	// Tracks the fields that we want to extract.
 	wanted fieldSet
+	// Tracks whether the current timestamp formes db2 format
```

Comment:
nit: formes seems out of places. "Tracks whether the current timestamp is of db2 format.", perhaps.